pandas/tests/arrays/test_datetimelike.py

```diff
@@ -140,6 +140,9 @@ def test_unbox_scalar(self):
         result = arr._unbox_scalar(pd.NaT)
         assert isinstance(result, (int, compat.long))
 
+    def test_scalar_from_string(self):
+        data = np.arange(10, dtype='i8') * 24 * 3600 * 10**9
+
         with pytest.raises(ValueError):
             arr._unbox_scalar('foo')
```

Comment:
this looks like I messed up a rebase and these last two lines belong in the previous test.  Ill try to fix in the next pass

Reply:
this is fixed in the rebase PR just made on your branch